ulauncher/modes/extensions/extension_remote.py

```diff
@@ -6,13 +6,13 @@
 from os.path import basename, getmtime, isdir
 from shutil import move, rmtree, which
 from tempfile import NamedTemporaryFile, TemporaryDirectory
-from typing import TypedDict
 from urllib.error import HTTPError, URLError
 from urllib.parse import urlparse
 from urllib.request import urlopen, urlretrieve
 
 from ulauncher.config import API_VERSION, paths
 from ulauncher.modes.extensions.extension_manifest import ExtensionIncompatibleRecoverableError, ExtensionManifest
+from ulauncher.utils.basedataclass import BaseDataClass
```

Comment:
Our own custom class. It is similar to TypedDict, but props can be accessed with the dot notation too (which I  think is much easier in code).